NickvisionTubeConverter.Shared/Models/VideoUrlInfo.cs

```diff
@@ -154,3 +184,32 @@ private VideoUrlInfo(string url)
         });
     }
 }
+
+/// <summary>
+/// A model of an available video resolution
+/// </summary>
+public class VideoResolution
```

Comment:
Sorry to be nit-picking but because `VideoResolution` class is used by `VideoInfo` class, it should be placed on top of that class and thus `VideoResoltion` should be the first class in the file.